comments/stakeholder_annotations.md

```diff
@@ -23,6 +23,41 @@ This may then be further aggregated as desired -- for example, into an equivalen
 
 ## Process
 
+### Defining Categories
+
+Annotation categories are defined per map from the map data sheets.
+A new "Annotation Categories" sheet is added with the following structure:
+
+| Name | Icon Url | Selected Icon Url | Help Text | Uses Mask | Description |
+|-------------|--------|----------|---------|---------|---------|
+| Interesting | //host.com/icon.svg | //host.com/inverse.svg | I find this interesting because... | true | Interesting category description |
+| Question | //host.com/icon.svg | //host.com/inverse.svg | I have a question... | 1 | Question category description |
+
+This is then transformed into RDF through the `annotation_categories` transform.
+
+E.g. output in Turtle format:
+```
+<http://visual-meaning.com/rdf/AnnotationCategory/interesting> a vm:AnnotationCategory ;
+    vm:description "Interesting category description" ;
+    vm:display "I find this interesting because…" ;
+    vm:hasAltIcon "//host.com/icon-inverse.svg" ;
+    vm:hasIcon "//host.com/icon.svg" ;
+    vm:name "Interesting" ;
+    vm:useMask "true" .
+```
+
+`Uses Mask` signifies whether icons should be used as a CSS masks, or as simple background images.
+CSS masks allow the icons to have branded colors, but come with the drawback that only the alpha values are considered when applying. This essentialy makes masked icons monochrome - opaque areas have the brand primary color applied, while fully transparent ones end up being white. Everything in between is a mix of primary and white.
+
+#### Maps with no categories
+
+Maps which do not need different annotation categories can still enable commenting by defining a single row in the "Annotation Categories" sheet.
```

Comment:
This supercedes the existing behaviour where setting any value in the `voting_map_path` field of the Map enables voting/commenting, which is a good change, as this behaviour doesn't make a huge amount of sense these days. I do wonder if it would be useful to retain some map-level master toggle for voting that could be used to open/close the voting process between certain dates, for example. Probably a separate piece of work to this though.